datafusion/src/dataframe.rs

```diff
@@ -145,7 +145,11 @@ pub trait DataFrame: Send + Sync {
     /// # Ok(())
     /// # }
     /// ```
-    fn union(&self, dataframe: Arc<dyn DataFrame>) -> Result<Arc<dyn DataFrame>>;
+    fn union(
```

Comment:
I think a new method e.g. `union_distinct` would make better sense - having booleans in APIs hurts readability.

alternatively - what about `df.union(df2)?.distinct()?` and not exposing a new method?